contrib/guix/manifest.scm

```diff
@@ -1,27 +1,27 @@
 (use-modules (gnu packages)
              (gnu packages autotools)
-             (gnu packages bash)
+             ((gnu packages bash) #:select (bash-minimal))
              (gnu packages bison)
-             (gnu packages certs)
-             (gnu packages cdrom)
-             (gnu packages cmake)
+             ((gnu packages certs) #:select (nss-certs))
+             ((gnu packages cdrom) #:select (xorriso))
+             ((gnu packages cmake) #:select (cmake-minimal))
              (gnu packages commencement)
              (gnu packages compression)
              (gnu packages cross-base)
              (gnu packages file)
              (gnu packages gawk)
              (gnu packages gcc)
-             (gnu packages installers)
+             ((gnu packages installers) #:select (nsis-x86_64))
              (gnu packages linux)
```

Comment:
Would it make sense to add `((gnu packages linux) #:select (util-linux linux-libre-headers-5.15))`?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/27811#issuecomment-1578197211).